src/PDFBuilder.js

```diff
@@ -450,9 +450,15 @@ export default class PDFBuilder extends WebformBuilder {
 
     const element = e.target;
     const type = element.getAttribute('data-type');
-
+    const key = element.getAttribute('data-key');
+    const group = element.getAttribute('data-group');
     const schema = fastCloneDeep(this.schemas[type]);
 
+    if (key && group) {
+      const info = this.getComponentInfo(key, group);
```

Comment:
This fix kind of seems like a hack to me. Can we discuss this fix in developer support meeting?